source/Waves.cpp

```diff
@@ -218,7 +218,7 @@ void Waves::setup(EnvCond *env, const char* folder)
 	if ((env->WaveKin == moordyn::WAVES_NONE) &&
 		(env->Current == moordyn::CURRENTS_NONE))
 	{
-		LOGMSG << "No Waves or Currents, or set externally";
+		LOGMSG << "No Waves or Currents, or set externally\n";
```

Comment:
endl better than \n

This is already fixed in https://github.com/mattEhall/MoorDyn/pull/31